test/library/packages/LinearAlgebra/correctness/testIdentity.good

```diff
@@ -0,0 +1 @@
+true
```

Comment:
Could you also test that it returns `false` for a non-identity matrix?

Reply:
Added test for this